infinispan-source.kamelet.yaml

```diff
@@ -0,0 +1,80 @@
+spec:
+  sources:
+    - content: |-
+        public class InfinispanWorkaround extends org.apache.camel.builder.RouteBuilder {
+          @Override
+          public void configure() throws Exception {
+            getContext().setAutowiredEnabled(false);
+          }
+        }
+      name: InfinispanWorkaround.java
+  definition:
+    title: "Infinispan Source"
+    description: |-
+      Get Events from an Infinispan cache
+    required:
+      - hosts
+      - username
+      - password
+      - hostAddresses
+    properties:
+      hosts:
+        title: Hosts
+        description: Specifies the host of the cache on Infinispan instance
+        type: String
+        default: 1000
```

Comment:
Default seems not right. Also you may want to specify if multiple hosts should be e.g. comma separaterd or so

Reply:
copy paste error. I'll fix it.